test/unittest/root/CMakeLists.txt

```diff
@@ -15,30 +15,29 @@
 if(GMOCK_FOUND)
     set(SRCS
         RootUnitTests.cpp
-        #DataReaderTests.cpp
         ../Common.cpp
-        ${PROJECT_SOURCE_DIR}/src/cpp/types/XRCETypes.cpp
-        ${PROJECT_SOURCE_DIR}/src/cpp/types/MessageHeader.cpp
-        ${PROJECT_SOURCE_DIR}/src/cpp/types/SubMessageHeader.cpp
-        ${PROJECT_SOURCE_DIR}/src/cpp/Root.cpp
         )
     add_executable(root_unit_test ${SRCS})
     add_gtest(root_unit_test
         SOURCES
             ${SRCS}
         DEPENDENCIES
+            micrortps_agent
```

Comment:
This change implies the test is not still a unittest. This test should be in other folder instead of be in `test/unittest`.

Reply:
Done!